data/tlg0057/tlg019/tlg0057.tlg019.opp-grc1.xml

```diff
@@ -63,19 +64,18 @@
 <listBibl xml:lang="lat">
  <biblStruct>
   <monogr>
-   <title>GALENI</title>
+   <title>Galeni De Victu Attenuante Liber</title>
    <editor>
 	<persName>
-	 <name xml:lang="lat">CAROLUS KALBFLEISCH</name>
+	 <name xml:lang="lat">Carolus Kalbfleisch</name>
 	</persName>
    </editor>
-   <author ref="urn:tmp">CAROLUS KALBFLEISCH</author>
+   <author ref="urn:tmp">Galenus</author>
```

Comment:
@AlisonBabeu Could you update this line so that @ref contains the URN for Galen, i.e., urn:cts:greekLit:tlg0057?